src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/RamDeliusAPIConvictionService.kt

```diff
@@ -41,13 +39,7 @@ class RamDeliusAPIConvictionService(
       .buildAndExpand(crn, id)
       .toString()
 
-    val auth = SecurityContextHolder.getContext()?.getAuthentication()
-    var token: JwtAuthenticationToken? = null
-    if (auth !== null) {
-      token = auth as JwtAuthenticationToken
-    }
-
-    val convictionDetails = ramDeliusApiClient.makeGetConvictionRequest(convictionPath, token)
+    val convictionDetails = ramDeliusApiClient.makeGetConvictionRequest(convictionPath, null)
```

Comment:
@awhodgson256 could you please default the null value in `ramDeliusApiClient.makeGetConvictionRequest(convictionPath, null)`

Reply:
Rectified that @kth13